### DISCUSSION TITLE
How to block Facebook but keep FB Messenger ?

### DISCUSSION THREAD

**Discussion by ficonni** [StevenBlack/hosts#1986] [Q&A]
Greetings, I would love to block Facebook but keep access to FB Messenger, is it possible and if so do you know by any chance which websites should not be put into the hosts file ?

Looking at `#Facebok ` lines there's plenty but none which seem to point at FB Messenger outisde of `messenger.com` but of course that's not enough when uncommenting all of the `Facebook` links to get it working.

Any help ?

**Comment by fglore6-prog** [comment]
> Greetings, I would love to block Facebook but keep access to FB Messenger, is it possible and if so do you know by any chance which websites should not be put into the hosts file ?
> 
> Looking at `#Facebok ` lines there's plenty but none which seem to point at FB Messenger outisde of `messenger.com` but of course that's not enough when uncommenting all of the `Facebook` links to get it working.
> 
> 
> Any help ?

**Comment by sontranhoangvn2-cyber** [comment]
Hi, my solution is to delete all the entries of # Facebook from the hosts file. Then, I only add 2 of these to block facebook:
0.0.0.0 facebook.com
0.0.0.0 www.facebook.com
I'm just a regular computer user so my method might not be the best
